addons/mail/static/src/models/thread/thread.js

```diff
@@ -426,6 +426,33 @@ function factory(dependencies) {
             this.update({ areAttachmentsLoaded: true });
         }
 
+        /**
+         * Fetch suggested recipients from the backend
+         */
+        async fetchAndUpdateSuggestedRecipients() {
+            const { [this.id]: data } = await this.async(() => this.env.services.rpc({
+                route: '/mail/get_suggested_recipients',
+                params: {
+                    model: this.model,
+                    res_ids: [this.id],
+                },
+            }));
+            if (!data) {
+                return;
+            }
+
+            const recipients = data.map(data => {
+                const recipient = this.env.models['mail.suggested_recipient_info'].convertData(data);
+                // Set the current thread on the recipient
+                recipient.thread = [['link', this]];
+                return recipient;
+            });
+
+            this.update({
```

Comment:
No newline.

You seem to use them a lot. Overuse kills the purpose of these newlines.

Purpose of empty newlines is to give some shape to code, to ease getting the structure of code just by a glance. Usually code inside a function should have no newlines.

Reply:
This is not how I use new line.

I use it to separate related code like : 

```
New var1
New var2

Some logic

Some actions

Some other logic

return stuff
```

Reply:
It works with small code, but with big code it adds too much newlines, and it becomes hard to catch the "shape".
Most of the time, code inside function is short and fulfill a certain goal, so there's no need for newlines.
Few exceptions are long functions which are best not being split in smaller functions (rare), and tests.

Reply:
This is a matter of preferences and how you like to read code. I have never had issue, even on big file.